aptos-move/framework/aptos-framework/doc/native_bridge.md

```diff
@@ -5,31 +5,37 @@
 
 
 
+-  [Struct `BridgeConfigRelayerUpdated`](#0x1_native_bridge_BridgeConfigRelayerUpdated)
+-  [Struct `BridgeFeeChangedEvent`](#0x1_native_bridge_BridgeFeeChangedEvent)
+-  [Struct `BridgeRiskDenominatorChangedEvent`](#0x1_native_bridge_BridgeRiskDenominatorChangedEvent)
+-  [Struct `BridgeInsuranceFundChangedEvent`](#0x1_native_bridge_BridgeInsuranceFundChangedEvent)
+-  [Struct `BridgeTransferInitiatedEvent`](#0x1_native_bridge_BridgeTransferInitiatedEvent)
+-  [Struct `BridgeTransferCompletedEvent`](#0x1_native_bridge_BridgeTransferCompletedEvent)
+-  [Resource `BridgeEvents`](#0x1_native_bridge_BridgeEvents)
 -  [Resource `AptosCoinBurnCapability`](#0x1_native_bridge_AptosCoinBurnCapability)
 -  [Resource `AptosCoinMintCapability`](#0x1_native_bridge_AptosCoinMintCapability)
 -  [Resource `AptosFABurnCapabilities`](#0x1_native_bridge_AptosFABurnCapabilities)
 -  [Resource `AptosFAMintCapabilities`](#0x1_native_bridge_AptosFAMintCapabilities)
--  [Struct `BridgeTransferInitiatedEvent`](#0x1_native_bridge_BridgeTransferInitiatedEvent)
--  [Struct `BridgeTransferCompletedEvent`](#0x1_native_bridge_BridgeTransferCompletedEvent)
--  [Resource `BridgeEvents`](#0x1_native_bridge_BridgeEvents)
 -  [Resource `Nonce`](#0x1_native_bridge_Nonce)
 -  [Resource `SmartTableWrapper`](#0x1_native_bridge_SmartTableWrapper)
 -  [Struct `OutboundTransfer`](#0x1_native_bridge_OutboundTransfer)
 -  [Resource `BridgeConfig`](#0x1_native_bridge_BridgeConfig)
--  [Struct `BridgeConfigRelayerUpdated`](#0x1_native_bridge_BridgeConfigRelayerUpdated)
--  [Struct `BridgeFeeChangedEvent`](#0x1_native_bridge_BridgeFeeChangedEvent)
 -  [Constants](#@Constants_0)
+-  [Function `initialize`](#0x1_native_bridge_initialize)
 -  [Function `normalize_u64_to_32_bytes`](#0x1_native_bridge_normalize_u64_to_32_bytes)
 -  [Function `is_inbound_nonce_set`](#0x1_native_bridge_is_inbound_nonce_set)
 -  [Function `create_details`](#0x1_native_bridge_create_details)
 -  [Function `add`](#0x1_native_bridge_add)
 -  [Function `set_bridge_transfer_id_to_inbound_nonce`](#0x1_native_bridge_set_bridge_transfer_id_to_inbound_nonce)
 -  [Function `assert_valid_bridge_transfer_id`](#0x1_native_bridge_assert_valid_bridge_transfer_id)
 -  [Function `bridge_transfer_id`](#0x1_native_bridge_bridge_transfer_id)
+-  [Function `bridge_relayer`](#0x1_native_bridge_bridge_relayer)
+-  [Function `insurance_fund`](#0x1_native_bridge_insurance_fund)
+-  [Function `risk_denominator`](#0x1_native_bridge_risk_denominator)
```

Comment:
The variable risk_denominator is not clear from the name and should be defined, but i am not certain where is best. Here is the text

>     The risk_denominator determines the fraction of the insurance fund that can be used for the per day budget for a given transfer direction.